### PR TITLE
Add support for firebug to Firefox browser profile

### PR DESCRIPTION
**Pull request by titusfortner**
This puts a small binary in the repository.

### CODE REVIEW

**Comment by jchang691** [comment]
Add Linux condition for this

**Comment by titusfortner** [comment]
This is updated with Linux condition and verified working. Should be ready to merge.

File: lib/watirmark/session.rb [hunk 1]
```diff
@@ -94,6 +94,15 @@ def default_firefox_profile
         profile['security.warn_viewing_mixed.show_once'] =  false
         profile['security.mixed_content.block_active_content'] = false
       end
+      if config.firebug
```
Comment (Blackbaud-TitusFortner) [inline lib/watirmark/session.rb]: Please push your changes

**Comment by Blackbaud-TitusFortner** [comment]
@kylemartinez  or @MatthewMazaika - Can one of you download this branch and verify it properly loads the extension for firefox?

**Pull request merged**